listenbrainz/config.py.sample

```diff
@@ -91,3 +91,6 @@ BETA_URL = 'https://api.listenbrainz.org'
 LISTENS_PER_GET = 100
 IMPORTER_LOG_FILE = '-'
 IMPORTER_DELAY = 3 # delay if requests don't return http 200
+
+
+LASTFM_PROXY_URL = 'http://0.0.0.0:8080/'
```

Comment:
What is this used for?

Reply:
When handshaking, the API returns a URL to be used by clients in the future, so we'd put `proxy.listenbrainz.org` in the config variable here in production and then clients would directly make requests to `proxy.listenbrainz.org`.

See this: https://github.com/metabrainz/listenbrainz-server/pull/248/files#diff-a3c56df132dfb50f5127034a2785c5bfR69